crates/net/network/src/config.rs

```diff
@@ -75,6 +75,10 @@ pub struct NetworkConfig<C> {
     #[cfg(feature = "optimism")]
     /// The sequencer HTTP endpoint, if provided via CLI flag
     pub sequencer_endpoint: Option<String>,
+    #[cfg(feature = "optimism")]
+    /// Whether to disable transaction gossip
+    #[cfg(feature = "optimism")]
+    pub disable_tx_gossip: bool,
```

Comment:
duplicated feature attributes
```suggestion
    /// Whether to disable transaction gossip
    #[cfg(feature = "optimism")]
    pub disable_tx_gossip: bool,
```